src/pocompiler.js

```diff
@@ -36,15 +43,15 @@ export function preparePoHeaders (headersRaw) {
     }
 
     return result;
-  }, {});
+  }, /** @type {Record<string, string>} */ ({}));
 }
 
 /**
  * Creates a PO compiler object.
  *
  * @constructor
- * @param {import('./types.js').GetTextTranslations} [table] Translation table to be compiled
- * @param {Partial<import('./types.js').parserOptions>} [options] Options
+ * @param {GetTextTranslations} [table] Translation table to be compiled
+ * @param {ParserOptions} [options] Options
```

Comment:
I capitalized it, also it doesn't really have to be marked as `Partial` it has no required properties.